src/dso_api/urls.py

```diff
@@ -1,17 +1,23 @@
 import django_healthchecks.urls
 from django.conf import settings
-from django.urls import include, path
+from django.urls import include, path, re_path
 from django.views.generic import RedirectView
 from rest_framework import exceptions
 
 import dso_api.dynamic_api.urls
 from dso_api.dynamic_api.oas3 import get_openapi_yaml_view
+from rest_framework_dso.views import multiple_trailing_slashes
 
 urlpatterns = [
     path("status/health/", include(django_healthchecks.urls)),
     path("v1/", include(dso_api.dynamic_api.urls)),
     path("v1/openapi.yaml", get_openapi_yaml_view(), name="openapi.yaml",),
     path("", RedirectView.as_view(url="/v1/"), name="root-redirect"),
+    re_path(
+        r"^v1/[A-Z,a-z,0-9]+/[A-Z,a-z,0-9]+/{2,}$",
```

Comment:
Ik denk dat hier hebben we meer nodig, `_` en `-` moeten er zijn. 
Maar ook  wat als alleen een naam zit er in? `/v1/parkeervakken//`?

Reply:
Had ik idd nog geen rekening mee gehouden, ik pas het aan :-)